src/sentry/workflow_engine/handlers/condition/tagged_event_handler.py

```diff
@@ -90,4 +93,17 @@ def evaluate_value(event_data: WorkflowEventData, comparison: Any) -> bool:
             if k.lower() == key or tagstore.backend.get_standardized_key(k) == key
         )
 
-        return match_values(group_values=tag_values, match_value=value, match_type=match)
+        result = match_values(group_values=tag_values, match_value=value, match_type=match)
+
+        logger.debug(
```

Comment:
this is just incase the top level logs don't show me everything, i want to be able to handle the user case(s) around this specific handler.